docs/arf.md

```diff
@@ -912,12 +912,12 @@ eID scheme.
 
 If all the legal and technical criteria have been met, including:
 
--   the certification of the Wallet Solution by accredited and
+- the certification of the Wallet Solution by accredited and
     designated CAB(s),
 
--   ex-ante supervisory activities by Member State supervisory bodies,
+- ex-ante supervisory activities by Member State supervisory bodies,
 
--   both the body providing EUDI Wallets and the ng PID Providers have
+- both the body providing EUDI Wallets and the PID Providers have
```

Comment:
I saw this fix in another PR so let's be careful when we merge so we avoid any conflicts